misc/navigation.py

```diff
@@ -1,116 +1,18 @@
 from talon.voice import Context, Key
-from os import system
-from talon.voice import Context, Key
 
 ctx = Context("navigation")
 
 keymap = {
-    # Requires activation of System Preferences -> Shortcuts -> Input Sources
-    # -> "Select the previous input source"
-    "change language": Key("ctrl-space"),
-    # Application navigation
-    "new window": Key("cmd-n"),
-    "(next window | gibby)": Key("cmd-`"),
-    "(last window | shibby)": Key("cmd-shift-`"),
-    "next space": Key("cmd-alt-ctrl-right"),
-    "last space": Key("cmd-alt-ctrl-left"),
-    # deleting
-    # '(delete line)': Key('cmd-right cmd-backspace'),
-    "steffi": Key("alt-ctrl-backspace"),
```

Comment:
this is a command that doesn't do anything different to carmex for me, and does not appear on https://en.wikipedia.org/wiki/Table_of_keyboard_shortcuts, so I conflated it with carmex. There are a few like that. I assume they are application-specific, but I don't know where to add them.